### PR TITLE
Issue #12: Make time outs configurable.

### PR DESCRIPTION
**Pull request by Jelle-S**

### CODE REVIEW

File: src/AbstractRoboFile.php [hunk 1]
```diff
@@ -1315,6 +1317,30 @@ protected function getLocalSettings($app = null, $timestamp = null)
         return $this->tokenReplace($this->getConfig()->get('local'), $replacements) + $defaults;
     }
 
+    protected function getTimeoutSetting($setting = null)
```
Comment (daften) [inline src/AbstractRoboFile.php]: This needs documentation on how to overrule this.

Comment (daften) [inline src/AbstractRoboFile.php]: Shouldn't default settings be in the property files?

**Review by daften — CHANGES_REQUESTED** [review]

File: src/AbstractRoboFile.php (hunk 1 above)
Comment (Jelle-S) [reply src/AbstractRoboFile.php]: They are:
`$timeoutSettings = $this->getConfig()->get('timeouts', []) + ...`

Comment (Jelle-S) [reply src/AbstractRoboFile.php]: as in, how to overrule in the property files? Or as in http://php.net/manual/en/language.oop5.inheritance.php

Comment (daften) [reply src/AbstractRoboFile.php]: First one :)

Comment (Jelle-S) [reply src/AbstractRoboFile.php]: Added a docblock. Real documentation should probably get fixed in #3 someday...

**Review by daften — APPROVED** [review]

**Pull request merged**